braindecode/regressor.py

```diff
@@ -250,3 +252,17 @@ def predict_proba(self, X):
             return y_pred.mean(-1)
         else:
             return y_pred
+
+    def predict_trials(self, X):
+        """Return trialwise predictions and targets.
+
+        Parameters
+        ----------
+        X : braindecode.datasets.BaseConcatDataset
+            A braindecode dataset to be predicted.
+
+        Returns
+        -------
+        trial_predictions, trial_labels: tuple(np.ndarray, np.ndarray)
+        """
+        return predict_trials(self.module, X)
```

Comment:
same here for comments above...  Maybe we could also consider to have a superclass EEGNeuralNet for both to avoid duplication?

Reply:
Yes, definitely. I just added one more duplicate to the large list of code duplicates in those classes...